electron/initStorage.ts

```diff
@@ -0,0 +1,34 @@
+import AddressBookStorage from './storage/AddressBookStorage'
+import AccountSettingsStorage from './storage/AccountSettingsStorage'
+import { IpcMain } from 'electron/main'
+import AbstractStorage from './storage/AbstractStorage'
+
+import SortType from 'Types/SortType'
+import IStorage from 'Types/IStorage'
+import Entity from 'Types/Entity'
+
+const storages: Record<string, AbstractStorage<Entity>> = {
+  'address-book': new AddressBookStorage(),
+  'account-settings': new AccountSettingsStorage(),
+}
+
+function initStorageCallbacks(ipcMain: IpcMain) {
+  Object.entries(storages).forEach(storage => {
+    ipcMain.handle(storage[0] + '-list', (e, searchTerm, sort) =>
+      storage[1].list(searchTerm, sort)
+    )
```

Comment:
```suggestion
  Object.entries(storages).forEach(([storageName, storage])=> {
    ipcMain.handle(storageName + '-list', (e, searchTerm, sort) =>
      storage.list(searchTerm, sort)
    )
```
Minor just to improve readability